userspace/su/jni/Android.mk

```diff
@@ -3,5 +3,9 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 LOCAL_MODULE := su
 LOCAL_SRC_FILES := su.c
+include $(BUILD_EXECUTABLE)
 
+include $(CLEAR_VARS)
+LOCAL_MODULE := su2
```

Comment:
I checked the CI result and i think we should change the module name to `simple_su`, `su` and `su2` is confusing!